app/models/vote.rb

```diff
@@ -0,0 +1,21 @@
+class Vote < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :review
+
+  validates_presence_of :user
+  validates_presence_of :review
+  validates :user_id, uniqueness: { scope: :review_id, :message => "User can only vote once per review."}
```

Comment:
Line is too long. [105/80]<br>Use the new Ruby 1.9 hash syntax.<br>Space inside } missing.